content/shmem_calloc.tex

```diff
@@ -23,18 +23,19 @@
   If the allocation succeeds, the pointer returned shall be suitably
   aligned so that it may be assigned to a pointer to any type of object.
   If the allocation does not succeed, or either \VAR{count} or \VAR{size} is
-  \CONST{0}, the return value is a null pointer.
+  \CONST{0}, the return value is a \const{NULL} pointer.
```

Comment:
This change is wrong. The text should read "the return value is a null pointer".